### PR TITLE
Release v1.0.8 1

### PR DESCRIPTION
**Pull request by nathan-at-least**

### CODE REVIEW

**Comment by ebfull** [comment]
ACK!

Note we decided *not* to change the protocol version in this PR.

File: doc/authors.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Daniel Cousens (4)
 lpescher (3)
 Pavel Janík (3)
 Alfie John (3)
+str4d (2)
```
Comment (daira) [inline doc/authors.md]: Should be added as an alias for Jack Grigg. Does not block.

Comment (bitcartel) [inline doc/authors.md]: Not captured above? @str4d

File: doc/man/zcashd.1 [hunk 2]
```diff
@@ -382,7 +382,7 @@ Generate coins (default: 0)
 .HP
 \fB\-genproclimit=\fR<n>
 .IP
-Set the number of threads for coin generation if enabled (\fB\-1\fR = all
+Set the number of threads for coin generation if enabled (\fB\fR = all
```
Comment (daira) [inline doc/man/zcashd.1]: Is this change correct?

File: doc/authors.md (hunk 1 above)
Comment (bitcartel) [inline doc/authors.md]: Ok I see github commit used str4d as name.

File: doc/man/zcashd.1 (hunk 2 above)
Comment (nathan-at-least) [inline doc/man/zcashd.1]: I don't know and suspect it is incorrect. I don't know yet how to verify this, hang on…

**Comment by ageis** [comment]
ACK

**Comment by nathan-at-least** [comment]
r+

**Comment by nathan-at-least** [comment]
Hey, yo, @zkbot r+

**Pull request merged**